src/message_channel.rs

```diff
@@ -247,6 +247,20 @@ where
     }
 }
 
+/// Functions which apply to any kind of [`MessageChannel`], be they strong or weak.
```

Comment:
For this comment to be true, you'd need to move away from the defaulted `Rc` variable and actually make this `impl` block generic over all `Rc`s :)

At the moment, it is only available for `Strong`.

Reply:
Ah yes, thanks for catching that! I updated the `impl` block accordingly.